autorun.ps1

```diff
@@ -0,0 +1,16 @@
+param (
+    [ValidateNotNullOrEmpty()][string]$SettingsFile = "$(Join-Path $Env:LOCALAPPDATA (Join-Path 'Raw Accel' 'settings.json'))"
+)
+
+$SettingsFile = Convert-Path($SettingsFile)
+$settingsDirectory = [IO.Path]::GetDirectoryName($SettingsFile)
+$settingsFileName = [IO.Path]::GetFileName($SettingsFile)
+
+$rawaccel_exe = Join-Path $PSScriptRoot rawaccel.exe
+$writer_exe = Join-Path $PSScriptRoot writer.exe
+
+if (Test-Path -PathType 'Container' -Path $settingsDirectory) {
+    Start-Process -WorkingDirectory $settingsDirectory -FilePath $writer_exe -ArgumentList $settingsFileName
+} elseif (New-Item -ItemType 'directory' -Path $settingsDirectory -Force) {
+    Start-Process -WorkingDirectory $settingsDirectory -FilePath $rawaccel_exe
+}
```

Comment:
Rather than initializing a whole new set of settings if they don't exist, what if we just read default settings from the install directory?

The settings file would still be created per-user once they opened Raw Accel using the shortcut, but not beforehand.

(credit to [#lysvero](https://discord.com/channels/755519723281711139/755519723281711142/1262151828838285364) for bringing this up)

Reply:
Because `rawaccel.exe` does not create directories, we would instead need to point the shortcut to a `rawaccel.exe` wrapper to make those directories. Altering the working directory *and* requiring a wrapper is a bit too convoluted for my taste.

Just creating the folder if it's not already there seems a lot more straightforward for this change. And if we're already creating the folder, why not also initialize the settings too.